tests/ha/clvmd_lvmlockd.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure clvmd or lvmlockd
+# Maintainer: Loic Devulder <ldevulder@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use version_utils qw(is_sle sle_version_at_least);
+use utils 'zypper_call';
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    my $lvm_conf = '/etc/lvm/lvm.conf';
+    my $lock_mgr = undef;
+
+    # lvmlockd is only available in SLE15+
+    if (get_var("USE_LVMLOCKD")) {
+        die 'lvmlockd can be only used on SLE15+' unless (is_sle && sle_version_at_least('15'));
+        $lock_mgr = 'lvmlockd';
+    }
+    else {
+        $lock_mgr = 'clvm';
+    }
```

Comment:
Is there another possible use for USE_LVMLOCKD? Or to rephrase the question: why don't determine the value of `$lock_mgr` depending on the version of SLES being tested?

Reply:
It's because in SLE15 we need to test both version, clvmd and lvmlockd. Lvmlockd will be the default version in SLE15, but clvmd will still be shipped for compatibility and migration purposes, so we still need to test it on SLE15.

In SLE < 15 there is only clvmd, so no need to use a variable, and I want to keep clvmd as the default test (for now). This let me to have the same test suites declaration in o.s.d. for SLE12 and SLE15. And I will add a new one for lvmlockd.

Reply:
Ah, understood. In that case, LGTM

Reply:
you can simplify little bit. 
in line 23 just write 
`my $lock_mgr='clvm'`

and drop else section at all

Reply:
@asmorodskyi, you're right, I should have seen this before submitting my PR, but I think a made too much change in the code ;-)